app/Transformers/TaskSchedulerTransformer.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+
+namespace App\Transformers;
+
+
+use App\Models\ScheduledJob;
+use App\Models\Scheduler;
+use App\Utils\Traits\MakesHash;
+
+class TaskSchedulerTransformer extends EntityTransformer
+{
+    use MakesHash;
+
+    protected $defaultIncludes = [
+        'job'
+    ];
+
+    public function includeJob(Scheduler $scheduler)
+    {
+        $transformer = new ScheduledJobTransformer($this->serializer);
+
+        return $this->item($scheduler->job, $transformer, ScheduledJob::class);
+    }
+
+    public function transform(Scheduler $scheduler)
+    {
+        return [
+            'id' => $this->encodePrimaryKey($scheduler->id),
```

Comment:
please review existing transformers how we delivery these, we type hint all properties